sites/header/header-docs-en.html

```diff
@@ -0,0 +1,302 @@
+---
+{
+	"title": "Header",
+	"language": "en",
+	"altLangPage": "header-docs-fr.html",
+	"secondlevel": false,
+	"dateModified": "2022-05-25",
+	"share": "true"
+}
+---
+<div class="wb-prettify all-pre hide"></div>
+
+<dl class="horizontal">
+	<dt>Status</dt>
+	<dd>Stable</dd>
+	<dt>Type</dt>
+	<dd>Canada.ca site functionality</dd>
+	<dt>Last review</dt>
+	<dd>2022-05-25</dd>
+</dl>
+
+<h2>Purpose</h2>
+<p>Displays the portion of the header section that's underneath the banner.</p>
+<p>Can contain the following components:</p>
+<ul>
+	<li>GCWeb Menu</li>
+	<li>Authentication</li>
+</ul>
+
+<h2>Working example</h2>
+<ul>
+	<li><a href="/templates/content-en.html">Content page including Header v3.0</a></li>
+	<li><a href="/sites/authentication/contextual-signin-en.html">Content page including Header v3.0 and Authentication section</a></li>
+</ul>
+
+<h2>How to implement</h2>
+
+<p>This version requires the implementation of the <a href="../gcweb-menu/gcweb-menu-docs-en.html">gcweb-menu version 2.0</a> where the <code>&lt;div class="container"&gt;</code> has been relocated in the header so outside the "gcweb-menu" component.</p>
+
+<h2>Evaluation and report</h2>
+<p>There is no evaluation and report available for this component.</p>
+
+<h2>API (Version 2.0)</h2>
+<table class="table table-bordered">
+	<tr>
+		<th>CSS Class</th>
+		<th>Template</th>
+		<th>Visual rendering</th>
+		<th>Schema</th>
+	</tr>
+	<tr>
+		<td>n.a.</td>
+		<td>Version 3.0</td>
+		<td>Version 1.0</td>
+		<td>n.a.</td>
+	</tr>
+</table>
+
+<h2>Before and After</h2>
```

Comment:
Je mettrais le titre "Before and After" en tant que H3 en-dessous d'un nouveau H2 nommé "Template". De ce fait, le titre "Previous versions" doit aussi devenir un H3 toujours sous le "Template", puis "Working examples" deviendra un H4. (Faire de même pour le Français)